app/src/main/res/layout/item_product_category.xml

```diff
@@ -1,77 +1,91 @@
 <?xml version="1.0" encoding="utf-8"?>
 <layout>
+
     <data>
+
         <variable
             name="product"
             type="com.example.internetshop.model.data.viewStates.ProductViewState" />
     </data>
-<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:app="http://schemas.android.com/apk/res-auto"
-    xmlns:tools="http://schemas.android.com/tools"
-    android:id="@+id/itemSimpleProdId"
-    android:layout_width="match_parent"
-    android:layout_height="wrap_content"
-    android:layout_gravity="center_horizontal"
-    android:background="@color/white"
-    android:onClick="@{()->product.onClick(product.id)}"
-    android:clickable="true">
 
-    <ImageView
-        android:id="@+id/productImage"
-        android:layout_width="@dimen/dimen_item_product_category_image_size"
-        android:layout_height="@dimen/dimen_item_product_category_image_size"
-        app:image_uri="@{product.imageUrl}"
-        tools:src="@drawable/photo"
-        app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toTopOf="parent" />
-
-    <TextView
-        android:id="@+id/numberOfReviews"
-        android:layout_width="wrap_content"
+    <androidx.cardview.widget.CardView xmlns:android="http://schemas.android.com/apk/res/android"
+        xmlns:app="http://schemas.android.com/apk/res-auto"
+        xmlns:tools="http://schemas.android.com/tools"
         android:layout_height="wrap_content"
-        android:textColor="@color/item_product_category_number_of_review_text_color"
-        android:textSize="@dimen/dimen_item_product_category_number_of_review_text_size"
-        android:text="@{product.numberOfReviews}"
-        app:layout_constraintBottom_toBottomOf="@+id/ratingBar"
-        app:layout_constraintStart_toEndOf="@+id/ratingBar"
-        app:layout_constraintTop_toTopOf="@+id/ratingBar"
-        tools:text="(10)" />
+        android:layout_width="match_parent"
+        app:cardCornerRadius="@dimen/dimen_item_category_card_corner_radius"
+        app:cardElevation="6dp">
+        <androidx.constraintlayout.widget.ConstraintLayout
+            android:id="@+id/itemSimpleProdId"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_gravity="center_horizontal"
+            android:background="@color/white"
+            android:clickable="true"
```

Comment:
do we need this clickable attribute? As i rembemer onClick works even without this attr